test/gatekeeper/test-mutations.yaml

```
@@ -0,0 +1,35 @@
+---
+apiVersion: mutations.gatekeeper.sh/v1beta1
+kind: Assign
+metadata:
+  name: allow-privilege-escalation
+spec:
+    match:
+      scope: Namespaced 
```

```suggestion
      scope: Namespaced
```
Remove whitespace at EOL